code/_onclick/hud/alert.dm

```diff
@@ -200,5 +200,4 @@ Reharging stations are available in robotics, the dormitory's bathrooms. and the
 		return usr.client.Click(master, location, control, params)
 
 /obj/screen/alert/Destroy()
-	PlaceInPool(src)
```

Comment:
That is odd. ya, I did. hmm, Fixing.

I did most of the destroy review/rework while drunk, high, and running on fumes.